editoast/src/views/infra/railjson.rs

```diff
@@ -129,19 +148,31 @@ async fn get_railjson(infra: Path<i64>, db_pool: Data<DbConnectionPool>) -> Resu
         .body(railjson))
 }
 
-#[derive(Debug, Clone, Deserialize)]
+/// Represents the query parameters for a POST request to /railjson.
```

Comment:
This comment isn't required since the struct is inlined as it won't show in the OpenApi.

```suggestion
/// Represents the query parameters for a `POST /infra/railjson` request
```